otelcollector/configmapparser/prometheus-config-merger-with-operator.rb

```diff
@@ -352,6 +357,68 @@ def populateDefaultPrometheusConfig
       end
     end
 
+    if !ENV["AZMON_PROMETHEUS_NETWORKOBSERVABILITYRETINA_SCRAPING_ENABLED"].nil? && ENV["AZMON_PROMETHEUS_NETWORKOBSERVABILITYRETINA_SCRAPING_ENABLED"].downcase == "true"
+      networkobservabilityRetinaMetricsKeepListRegex = @regexHash["NETWORKOBSERVABILITYRETINA_METRICS_KEEP_LIST_REGEX"]
+      networkobservabilityRetinaScrapeInterval = @intervalHash["NETWORKOBSERVABILITYRETINA_SCRAPE_INTERVAL"]
+      if (isConfigReaderSidecar || currentControllerType == @replicasetControllerType)
+        #do nothing -- kappie is not supported to be scrapped automatically outside ds. if needed, customer can disable this ds target, and enable rs scraping thru custom config map
```

Comment:
[nit] Kappie will should be renamed to Retina. Same applies for the other sections. Thanks!

Reply:
Is it alright if we handle this nit when transition is complete, and we remove all traces of the Kappie name from ama? the manual testing process is very involved for a new image+commit